package.json

```diff
@@ -66,16 +68,18 @@
     "typescript": "^5.0.2"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
+    "start": "vite --port 3000",
```

Comment:
hardcoding port number not necessary? when running couple project locally, 3000 will be in conflict anyways